File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        
        listS = s.split(" ")
        temp = []
        print(listS)
        
        for char in listS:
            if char.isalpha():
                temp.append(char)
                
                
        
        return len(temp[-1])
        

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        
        String[] listS = s.split(" ");
        
        List<String> temp = new ArrayList<>();
        
        for (int i=0; i< listS.length; i++) {
            if (!listS[i].equals(" ")) {
                temp.add(listS[i]);
                    }   
                }
        if (!temp.isEmpty()) {
            return temp.get(temp.size() - 1).length();
        } else {
            return 0; // If no valid words are found
        }
            }
        }